spec/features/admin/sort_view_in_organisation_list_spec.rb

```diff
@@ -0,0 +1,34 @@
+describe 'sorting the columns in the organisation list' do
+  context 'when admin views the names of organisations' do
+    let!(:organisation_1) { create(:organisation, name: "ABC", created_at: Date.yesterday) }
```

Comment:
I'd use real-looking data in acceptance specs, as they're often the first time a reader will encounter concepts, and it helps explain them better.

```suggestion
    let!(:organisation_1) { create(:organisation, name: "Department of Silly Hats", created_at: Date.yesterday) }
```